File: C/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int height;

    do
    {
        printf("What is the height?\n");
        height = get_int();
    }
    while (height < 0 || height > 23);

// For each row
    for (int row = 0; row < height; row++)
    {

// Left side spaces
        for (int spaces = 0; spaces < height-row-1; spaces++)
        {
            printf(" ");
        }

// Left side hashes
        for (int hashes = 0; hashes <= row; hashes++)
        {
            printf("#");
        }

// Print gap
        printf("  ");


// Right side hashes
        for (int hashes = 0; hashes <= row; hashes++)
        {
            printf("#");
        }

// Print new line
        printf("\n");


    }
}

File: Python/mario.py
from cs50 import get_int

# Validate user input
while True:
    height = get_int("What is the height?")
    if height >= 0 and height <= 23:
        break

# Draw pyramid
for i in range(height):
    print(" " * (height - i - 1), end="")
    print("#" * (i + 1), end="")
    print("  ", end="")
    print("#" * (i + 1))
